en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md

```diff
@@ -1,43 +1,51 @@
 # Choreo Connect Overview
 
-Choreo Connect is an API Gateway for micro services, which is cloud-native, decentralized and developer centric.
+Choreo Connect (the API Microgateway) is an API Gateway for microservices, which is cloud-native, decentralized, and developer-centric.
 
-The WSO2 Choreo Connect is a lightweight message processor for APIs. The Choreo Connect is used for message security, transport security, routing, and other common API Management related quality of services.
+WSO2 Choreo Connect is a lightweight message processor for APIs. You would need to use WSO2 Choreo Connect for message security, transport security, routing, and other common API Management related quality of services (QoS).
 
-[![]({{base_path}}/assets/img/deploy/mgw/mgw_basic.png)]({{base_path}}/assets/img/deploy/mgw/mgw_basic.png)
+[![Choreo Connect Basic Architecture]({{base_path}}/assets/img/deploy/mgw/mgw_basic.png){: style="width:90%" }]({{base_path}}/assets/img/deploy/mgw/mgw_basic.png)
 
 ## Choreo Connect Components
 
 Choreo Connect consists of the following three components.
 
-### 1. Router
+- [Router](#router)
+- [Enforcer](#enforcer)
+- [Adapter](#adapter)
 
+<<<<<<< HEAD
```

Comment:
Looks like merge conflcts.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/5032/commits/f02f25aa5ccc060afcd096f91bb1273a3fc042a8